src/components/user/order/OrderAccountInfo.tsx

```diff
@@ -0,0 +1,114 @@
+import AppLabel from '@components/common/label/AppLabel';
+import styled from '@emotion/styled';
+import { Color } from '@resources/colors';
+import { colFlex, rowFlex } from '@styles/flexStyles';
+import React, { useEffect } from 'react';
+import useWorkspace from '@hooks/user/useWorkspace';
+import { useSearchParams } from 'react-router-dom';
+import { Account } from '@@types/index';
+import { defaultAccountValue } from '@@types/defaultValues';
+
+const Container = styled.div`
+  width: 100%;
+  gap: 5px;
+  ${colFlex({ justify: 'center', align: 'center' })}
+`;
+
+const TitleContainer = styled.div`
+  width: 87%;
+  ${rowFlex({ justify: 'space-between', align: 'center' })}
+`;
+
+const AccountInfo = styled.div`
+  width: 100%;
+  font-size: 13px;
+  font-weight: 500;
+  color: #898989;
+  padding: 10px 20px;
+  box-sizing: border-box;
+  background: ${Color.LIGHT_GREY};
+  ${colFlex({ justify: 'center' })}
+`;
+
+const CopyButton = styled.button`
+  width: 90px;
+  height: 23px;
+  font-size: 12px;
+  background: ${Color.WHITE};
+  border: 1px solid ${Color.GREY};
+  border-radius: 20px;
+  ${rowFlex({ justify: 'center', align: 'center' })}
+`;
+
+const InfoRow = styled.div`
+  width: 100%;
+  gap: 5px;
+  ${rowFlex({ justify: 'center', align: 'center' })}
+`;
+
+const Key = styled.div`
+  ${rowFlex({ justify: 'end', align: 'center' })}
+`;
+
+const Value = styled.div`
+  width: 250px;
+  ${rowFlex({ justify: 'start', align: 'center' })}
+`;
+
+const Divider = styled.span``;
+
+function OrderAccountInfo() {
+  const [searchParams] = useSearchParams();
+  const workspaceId = searchParams.get('workspaceId');
+  const [accountInfo, setAccountInfo] = React.useState<Account>(defaultAccountValue);
+
+  const { fetchWorkspaceAccount } = useWorkspace();
+
+  const getAccountInfo = async () => {
+    const account = await fetchWorkspaceAccount(workspaceId);
+    if (!account) return;
+
+    setAccountInfo(account);
+  };
+
+  const copyAccountInfo = () => {
+    const bankName = accountInfo.bank.name;
+    const accountNumber = accountInfo.accountNumber;
```

Comment:
```suggestion
    const bankName = accountInfo?.bank?.name;
    const accountNumber = accountInfo?.accountNumber;
```
여기도 아래와 같이 ? 를 추가해줘야 할 것 같아요!